biologging-sensor-data/package.json

```diff
@@ -23,7 +23,8 @@
     "react": "^18.2.0",
     "react-chartjs-2": "^5.2.0",
     "react-dom": "^18.2.0",
-    "react-leaflet": "^4.2.1"
+    "react-leaflet": "^4.2.1",
+    "react-router-dom": "^6.22.3"
```

Comment:
Why is this package here?

Reply:
@zrohacova , which package? Are you asking about the 'react-router-dom'?

Reply:
Yes